tests/partitioning/test.yaml

```diff
@@ -55,3 +55,13 @@ actions:
     description: Compare expected and actual
     chroot: false
     command: bash -c 'diff -u <(jq . ${RECIPEDIR}/expected.json) <(jq . ${RECIPEDIR}/actual.json)'
+
+  # Verify that postprocess commands have cwd set to artifact directory
+  # by checking if test.img exists in the current directory and that
+  # the current directory equals $ARTIFACTDIR.
+  # This test is run with --artifactdir=run so the artifact directory
```

Comment:
utter nitpick: This is a lie as its with run with artifactdir=scratch.

wonder if to be sure we should enforce in this test that ARTIFACTDIR != RECIPEDIR

Reply:
good catch; i updated the test